lib/screens/edit_product_screen.dart

```diff
@@ -134,12 +102,16 @@ class _EditProductScreenState extends State<EditProductScreen> {
     return null;
   }
 
-  String validatorTitle(String value) => value.isEmpty ? 'Please provide a value' : null;
+  String validatorTitle(String value) {
+    if (value.isEmpty) {
+      return 'Please provide a value.';
+    }
+    return null;
```

Comment:
Do something like this 

`return value.isEmpty ? 'Error description example' : null`